### DISCUSSION TITLE
Beware before you pay.!

### DISCUSSION THREAD

**Discussion by pranavateja99** [shivaya-dav/DogeRat#79] [General]
Shiv yadav and his friend manish wont respond once you pay them, you will get same youtube premium apk which work as normal youtube with ads. Not apk of your choice or even something like calculator. its useless. They just ignore.  Not sure what these people think of android users, dumb? who cant differentiate youtube free/ youtube premium and app called youtube premium which works as normal youtube with ads and yet ask all permissions. Till now i asked them many times to provide something like filemanager or something like calculator. They just ignore or cant capable of it.

**Comment by heatherlinda** [comment]
As I sifted through the evidence provided by Hi Tech Recovery, my suspicions were confirmed. Some messages and conversations raised red flags and shattered the trust I had once placed in my husband. It was a painful realization, but one that I knew I needed to confront. The evidence didn't stop at messages alone. Hi Tech Recovery also uncovered multimedia files that painted a vivid picture of my husband's actions. From incriminating photos to compromising videos, it was undeniable proof of his infidelity. It was like being hit with a one-two punch in the gut and heart. Coping with the aftermath of infidelity is no easy task, but with a little bit of magic, you can begin the healing process. It is with the help of Hi Tech Recovery that I was able to see through his phone that was hacked without his knowledge that she derived pleasure in other men. This is a very sad write up to me but it is better because I now know where I stand. Reach out to Hi Tech Recovery for more information on 
Email:  hitechrecovery(At)contractor.net  and Telegram: @hitechrecovery
